README.md

```
@@ -382,12 +380,20 @@ Activate ingress support:
 $ minikube addons enable ingress
 ```
 
+```bash
+# for recent (1.32+) minikube versions, need to relax ingress-nginx security:
+$ kubectl -n ingress-nginx patch configmap ingress-nginx-controller   --type merge   -p '{"data":{"allow-snippet-annotations":"true","annotations-risk-level":"Critical"}}'
+$ kubectl -n ingress-nginx rollout restart deployment ingress-nginx-controller
+```
+
 Verify everything is ok with:
 ```bash
 $ minikube status
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)